### PR TITLE
fix: Changing schema to public

### PR DESCRIPTION
**Pull request by ptiurin**

### CODE REVIEW

File: tests/integration/test_sqlalchemy_integration.py [hunk 1]
```diff
@@ -58,6 +58,8 @@ def test_has_table(self, engine: Engine, fact_table_name: str):
     def test_get_table_names(self, engine: Engine):
         results = engine.dialect.get_table_names(engine)
         assert len(results) > 0
+        results = engine.dialect.get_table_names(engine, "public")
+        assert len(results) > 0
```
Comment (stepansergeevitch) [inline tests/integration/test_sqlalchemy_integration.py:62]: Can we also add a test for empty results?

**Review by stepansergeevitch — APPROVED** [review]
lgtm, single comment

File: tests/integration/test_sqlalchemy_integration.py (hunk 1 above)
Comment (ptiurin) [reply tests/integration/test_sqlalchemy_integration.py:62]: good idea, added.
